pkg/operators/v1/olmconfig_types.go

```diff
@@ -0,0 +1,71 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	DisableCopiedCSVsConditionType = "DisableCopiedCSVsConfiguration"
```

Comment:
Should this be past tense so that it reads a little more like some state has (or hasn't) been achieved? Not sure if this was captured in the EP. If so, disregard. E.g.:
```yaml
conditions:
- type: DisabledCopiedCSVs
  status: True
```

Or:

```yaml
conditions:
- type: DisabledCopiedCSVs
  status: False
  reason: DeleteCSVFailure
  message: 'delete csv "foo" in namespace "bar": not permitted'
```

Reply:
This seems fine and was not captured in the EP

Reply:
Addressed in the latest version of the PR. Note that it isn't possible to discern the state of the condition without both the `status` and `reason`:

`status` is false because copied csvs are enabled:
```yaml
conditions:
- type: DisabledCopiedCSVs
  status: False
  reason: CopiedCSVs enabled
  message: CSVs are being copied
```

vs.

`status` is false because not all copied csvs were deleted:
```yaml
conditions:
- type: DisabledCopiedCSVs
  status: False
  reason: DeleteCSVFailure
  message: 'delete csv "foo" in namespace "bar": not permitted'
```

Not really an issue, but something worth calling out in the eventual docs.